util/helm/helm_test.go

```diff
@@ -166,6 +166,7 @@ func TestHelmArgCleaner(t *testing.T) {
 		`not, clean`: `not\, clean`,
 		`a\,b,c`:     `a\,b\,c`,
 		`{a,b,c}`:    `{a,b,c}`,
+		`,,,,,\,`:    `\,\,\,\,\,\,`,
```

Comment:
In the current code, if you type `\\,`, it will still come out as `\\,` because we don't append a \ to commas that are already escaped.

But maybe I'm misunderstanding, is it correct that when I type `\\,` I get `\\\,`?

Reply:
Good question... we have the somewhat-weird behavior of only escaping commas if they don't _look_ already-escaped.

The scenario I'm describing is probably rare. I say we merge what you have and deal with the additional edge case if we ever hit it.

Reply:
Thank you for your explanation, i will aware of the issue. 😊